lib/events/Profile/receivedOfferFullCardEvents.js

```diff
@@ -14,12 +14,35 @@ Template.receivedOfferFullCard.events({
 		var options = {
 			id: id,
 			listingId: listing,
-			destination: Session.get('offerSelected').creator_id
+			destination: [Session.get('offerSelected').creator_id, Meteor.userId()]
```

Comment:
Line is too long.<br>Identifier 'creator_id' is not in camel case.<br>'Session' is not defined.<br>'Meteor' is not defined.